apps/editing-toolkit/editing-toolkit-plugin/help-center/class-help-center.php

```diff
@@ -78,7 +79,13 @@ public static function is_next_steps_tutorial_enabled() {
 	 * Enqueue Help Center assets.
 	 */
 	public function enqueue_script() {
-		$script_dependencies = $this->asset_file['dependencies'];
+		// Remove wp-preferences from the dependencies array because it is not available on the support page.
+		$script_dependencies = array_filter(
```

Comment:
I noticed that `wp-preferences` was breaking the dependencies for the support site, I believe we actually don't need it, and this is how I am ignoring it. I am open to suggestions on how improve it :)

Reply:
Hi @noahtallen! I remember you fought this issue before with react-i18n. Do you have any idea to make `wp-preferences` available?